src/diggler/context.d

```diff
@@ -199,42 +199,54 @@ struct Context
 
 		if(auto user = tracker.findUser(nickName))
 		{
-			if(user.payload.isAdmin)
+			if(user.payload.isIdentified)
 				return true;
 
-			auto curFiber = _bot.commandQueue.fiber();
-			bool result = false;
+			string accountName = getAccountName(nickName);
+			if(accountName is null)
+				return false;
 
-			void onWhoisAccountReply(in char[] nick, in char[] accountName)
+			return sortedAdminList.contains(Bot.Admin(nickName, cast(immutable)accountName));
```

Comment:
AFAICS, this will never run. If not `user.payload.isIdentified` then `accountName is null`.